### PR TITLE
feat: 🎸 Added link component for green-react

### PR DESCRIPTION
**Pull request by ethan-qsh**

### CODE REVIEW

**Comment by sebopensource** [comment]
👋 Thanks for creating a pull request! 🚀 Checkout the storybook we created for it [here](https://sebgroup.github.io/green/pr-289/chlorophyll/).

File: libs/react/src/lib/link/link.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  link?: string
+  text?: string
+}
+
+export const Link = ({ link, text }: LinkProps) => {
+  const props: HTMLProps<HTMLAnchorElement> = {}
+  if (link) props.href = link
+
+  return (
+    <div>
```
Comment (kherP) [inline libs/react/src/lib/link/link.tsx:13]: what is this div for?

File: libs/react/src/lib/link/link.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React, { HTMLProps } from 'react'
+
+interface LinkProps {
+  link?: string
+  text?: string
+}
```
Comment (kherP) [inline libs/react/src/lib/link/link.tsx:6]: probably we should extend the props from default anchor element props, to support accessibility. For instance,
<Link link="some link" text="Read more" aria-label="Read more about this link" />.

File: libs/react/src/lib/link/link.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+export const Link = ({ link, text }: LinkProps) => {
+  const props: HTMLProps<HTMLAnchorElement> = {}
+  if (link) props.href = link
+
+  return (
+    <div>
+      <a href={link} rel="noreferrer">
```
Comment (kherP) [inline libs/react/src/lib/link/link.tsx:14]: i dont think we should define `rel="noreferrer"` for the user

File: libs/react/src/lib/link/link.tsx (hunk 1 above)
Comment (ethan-qsh) [reply libs/react/src/lib/link/link.tsx:13]: No purpose, we just overlooked it. We will remove it in the next commit.

File: libs/react/src/lib/link/link.tsx (hunk 3 above)
Comment (ethan-qsh) [reply libs/react/src/lib/link/link.tsx:14]: Alright, will remove it in the next commit

File: libs/react/src/lib/link/link.tsx (hunk 2 above)
Comment (ethan-qsh) [reply libs/react/src/lib/link/link.tsx:6]: Alright, do mean to add more of the default anchor element props to the interface?

Comment (kherP) [reply libs/react/src/lib/link/link.tsx:6]: my suggestion is to extend React.HTMLProps<HTMLAnchorElement> but maybe you can omit the children and href, depends on how you want this component to work

**Pull request merged**